site2/website/versioned_docs/version-2.6.3/client-libraries-java.md

```diff
@@ -186,6 +186,7 @@ ProducerCryptoFailureAction|`cryptoFailureAction`|Producer should take action wh
 long|`batchingMaxPublishDelayMicros`|Batching time period of sending messages.|TimeUnit.MILLISECONDS.toMicros(1)
 int|batchingMaxMessages|The maximum number of messages permitted in a batch.|1000
 boolean|`batchingEnabled`|Enable batching of messages. |true
+BatcherBuilderType| `batcherBuilderType`| BatcherBuilder type <br/><br/>Two types are available:<li>Default</li><li>Key_Based</li>|BatcherBuilderType.Default
```

Comment:
you are modifying the wrong file, this is for version 2.6.3, please modify the "latest" version of the doc

Reply:
yikes. Reverted and applied to the correct file